### PR TITLE
Localizes permission labels

### PR DESCRIPTION
**Pull request by dkastl**
Changes proposed in this pull request:
- Localizes permission labels

Screenshot from 2021-11-05 00-47-00

### CODE REVIEW

**Review by sanak — COMMENTED** [review]
Well, about module name, there is already famous plugin `redmine_issue_template`, so let me check whether it will not conflict.

**Comment by dkastl** [comment]
OK. I just saw that the Japanese translation always said "Template". In other context it's sometimes called "Canned Response".

File: config/locales/ja.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ ja:
   label_text_block_text: テンプレート
   label_text_block_plural: テンプレート
   label_text_block_new: 新しいテンプレート
+  permission_view_text_blocks: "テンプレートの閲覧"
+  permission_manage_text_blocks: "テンプレートの管理"
+  project_module_text_blocks: "テンプレート"
```
Comment (sanak) [inline config/locales/ja.yml:10]: @dkastl Well, another plugin uses `チケットテンプレート` as project module name, so how about to use `コメントテンプレート` from the functionality ?
https://github.com/agileware-jp/redmine_issue_templates/blob/master/config/locales/ja.yml#L24

Other part can be kept `テンプレート` as it is.

Comment (dkastl) [reply config/locales/ja.yml:10]: Well, I'm not the right person to decide what sounds good in Japanese, so I just follow any suggestion. In English I'm using "Text blocks".

Comment (dkastl) [reply config/locales/ja.yml:10]: Or something like "Response template"?

We could also just extend the section headline and make it a linger name, but keep "manage" and "view" items shorter.

Comment (sanak) [reply config/locales/ja.yml:10]: @dkastl  
Well, I actually noticed that the issue templates plugin seems to support comment (note) template with `コメント用テンプレート` translation,
https://github.com/akiko-pusu/redmine_issue_templates/releases/tag/0.3.1
https://github.com/agileware-jp/redmine_issue_templates/blob/master/config/locales/ja.yml#L63

so, let's keep current our translation (`テンプレート`) as it is.
I approve this.

**Review by sanak — APPROVED** [review]

**Pull request merged**